backend/app/transcendence/.eslintrc.js

```diff
@@ -0,0 +1,327 @@
+module.exports = {
+    "env":
+    {
+        "browser": true,
+        "es2021": true
+    },
+    "extends":
+    [
+        "eslint:recommended",
+        "plugin:@typescript-eslint/recommended"
+    ],
+    "overrides":
+    [
+        {
+            "env": {
+                "node": true
+            },
+            "files": [".eslintrc.{js,cjs}"],
+            "parserOptions": {
+                "sourceType": "script"
+            }
+        }
+    ],
+    "parser": "@typescript-eslint/parser",
+    "parserOptions":
+    {
+        "ecmaVersion": "latest",
+        "sourceType": "module"
+    },
+    "plugins":
+    [
+        "@typescript-eslint"
+    ],
+    "rules":
+    {
+        // "indent":
+        // [
+        //     "error",
+        //     "tab"
+        // ],
+        // "linebreak-style": [
+        //     "error",
+        //     "unix"
+        // ],
+        "quotes":
+        [
+            "error",
+            "double"
+        ],
+        "semi":
+        [
+            "error",
+            "always"
+        ],
+        // custom rules based on 42
+        // POSSIBLE PROBLEMS
+        "no-await-in-loop": "error",
+        "no-class-assign": "error",
+        "no-compare-neg-zero": "error",
+        "no-cond-assign":
+        [
+            "error",
+            "always"
+        ],
+        "no-const-assign": "error",
+        "no-constant-condition": "error",
+        "no-constructor-return": "error",
+        "no-dupe-args": "error",
+        "no-dupe-class-members": "error",
+        "no-dupe-else-if": "error",
+        "no-dupe-keys": "error",
+        "no-duplicate-case": "error",
+        "no-duplicate-imports":
+        [
+            "error",
+            {
+                "includeExports": true
+            }
+        ],
+        "no-ex-assign": "error",
+        "no-fallthrough": 
+        [
+            "error",
+            {
+                "commentPattern": "break[\\s\\w]*omitted"
+            }
+        ],
+        "no-import-assign": "error",
+        "no-inner-declarations": "error",
+        "no-self-compare": "error",
+        "no-use-before-define": "error",
+        // SUGESTED
```

Comment:
suggested*

Reply:
Yes Splitted the conf object into two main part, bellow this line it is suggested recommandation for a good code quality

Reply:
I only explicited the typo in "suggested" written with one "g"